fvtest/threadextendedtest/processTimeTest.cpp

```diff
@@ -81,115 +45,126 @@ systemTimeCPUBurn(void)
 }
 
 /**
- * This function records process times after each call to the 'user time' CPU burner.
- * It also checks for valid return codes after each call.
- *
- * @param[in]  arrayLength		Length of the processTimeArray
- * @param[out] processTimeArray	Pointer to the array of process times
+ * Do work in the process consuming user CPU time. This is the core function to
+ * test the user time returned by get_process_times. Its run time is
+ * deterministic and it does the same amount of work each time it is invoked.
  */
 void
-runCPUBurnToTestUserTimes(omrthread_process_time_t *processTimeArray, uintptr_t arrayLength)
+userTimeCpuBurn()
 {
-	uintptr_t i = 0;
-	intptr_t retValOfGetProcessTimes = 0;
-
-	/*
-	 * If we increase the workload done (running the matrixSquare multiple times),
-	 * CPU time recorded should increase after each run
-	 */
-	for (i = 0; i < arrayLength; i++) {
-		matrixSquare();
-		retValOfGetProcessTimes = (omrthread_get_process_times(processTimeArray++) || retValOfGetProcessTimes);
-		EXPECT_TRUE(0 == retValOfGetProcessTimes);
+	for (size_t i = 0; i < 40000; i++) {
+		VM_AtomicSupport::nop();
 	}
 }
-/**
- * This function records process times after each call to the 'system time' CPU burner.
- * It also checks for valid return codes after each call.
- *
- * @param[in]  arrayLength		Length of the processTimeArray
- * @param[out] processTimeArray	Pointer to the array of process times
- */
-void
-runCPUBurnToTestSystemTimes(omrthread_process_time_t *processTimeArray, uintptr_t arrayLength)
-{
-	uintptr_t i = 0;
-	intptr_t retValOfGetProcessTimes = 0;
 
-	/*
-	 * If we increase the workload done (running the same function multiple times),
-	 * CPU time recorded above should increase after each run
-	 */
-	for (i = 0; i < arrayLength; i++) {
-		systemTimeCPUBurn();
-		retValOfGetProcessTimes = (omrthread_get_process_times(processTimeArray++) || retValOfGetProcessTimes);
-		EXPECT_TRUE(0 == retValOfGetProcessTimes);
+TEST(ThreadCpuTime, userCpuTimeIncreasesMonotonically)
+{
+	omrthread_process_time_t cpuTime, prevCpuTime;
```

Comment:
To make this code more clear would you place each declaration on a separate line?  There are other cases below that should also be fixed.